OneLogTrait.php

```diff
@@ -0,0 +1,93 @@
+<?php declare(strict_types=1);
+
+namespace KoderHut\OnelogBundle;
+
+/**
+ * Trait OneLogTrait
+ *
+ * @author Joao Jacome <969041+joaojacome@users.noreply.github.com>
+ */
+trait OneLogTrait
+{
+    /**
+     * @param mixed $message
+     * @param array $context
+     */
+    public function emergency($message, array $context = array())
+    {
```

Comment:
since all these methods should proxy to the default logger I would just call those methods on the default logger itself and avoid the dynamic method call in the log()